tests/vm_test.go

```diff
@@ -828,10 +831,11 @@ var _ = Describe("[rfe_id:1177][crit:medium][vendor:cnv-qe@redhat.com][level:com
 					Expect(err).ToNot(HaveOccurred())
 					Expect(newVM.Spec.RunStrategy).ToNot(BeNil())
 					Expect(*newVM.Spec.RunStrategy).To(Equal(v1.RunStrategyManual))
+					By("Ensuring stateChangeRequests list is cleared")
 					Expect(len(newVM.Status.StateChangeRequests)).To(Equal(0))
 				})
 
-				It("should stop", func() {
+				It("[test_id:2189] should stop", func() {
```

Comment:
I am missing the actions of start and stop, and wait for to be removed verify  vm's runStrategy = Manual Ensure vm.status.stateChangeRequests is not present in vm's manifest

Reply:
Confirmed test is correct in offline conversation with @ILpinto